Sources/Modules/GroupsModule.swift

```diff
@@ -164,12 +164,14 @@ public class GroupsModule {
     ///
     /// - Parameters:
     ///   - name: Only return groups whose name contains the given string (case insensitive).
```

Comment:
If `filterTerm` filters on name, then what does `name` do?  I'm thinking it's actually ignored by the API?  It's not listed in the API docs: https://developer.box.com/reference/get-groups/